etc/adminhtml/system.xml

```diff
@@ -40,6 +40,10 @@
                     <label>Label description</label>
                     <comment>This description will appear on the shipment label. The following parts can be used: %order_nr%, %delivery_date%, %product_id%, %product_name%, %product_qty%.</comment>
                 </field>
+                <field id="country_of_origin" translate="label comment" type="text" sortOrder="214" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Country of origin</label>
+                    <comment>This country will appear on the international consignment labels. This is where your products are shipped from. You can use "NL", "BE", "DE" etc.</comment>
```

Comment:
Zet in de command ook iets over het veld country of manufacture. Er kunnen namelijk vragen komen waarom er bijvoorbeeld geen NL zien is terwijl dat wel is ingevuld.